esp/platform/espcfg.cpp

```diff
@@ -361,6 +361,16 @@ CEspConfig::CEspConfig(IProperties* inputs, IPropertyTree* envpt, IPropertyTree*
         if (m_cfg->getProp("@daliServers", daliservers))
             initDali(daliservers.str()); //won't init if detached
 
+#ifndef _CONTAINERIZED
+        IPropertyTree * espConfigCost = m_envpt->queryPropTree("cost");
```

Comment:
"cost" doesn't seem to match the tag that was added to esp.xsl.

Reply:
The original environment.xml had the tag names upper-cased but the new yaml configs have the tags in camel case.  I used lower case "cost" tag for m_envpt, so that there wouldn't be unnecessary concern that m_envpt was lowercase whilst the tag in global is lowercase.

It works because it's case in-sensitive.  

I'll change this to match environment.xml